prow/config/jobs/istio.yaml

```diff
@@ -250,6 +250,23 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: " --istio.test.retries=1 "
 
+  - name: integ-cni-k8s-tests
```

Comment:
@bianpengyuan can you make this optional and we can see if it passes consistently? https://github.com/bianpengyuan/test-infra/blob/cni-test/prow/config/jobs/istio.yaml#L21

Reply:
Actually, nevermind. Talking to Eric, I think that the hidden argument makes it optional as well.

Reply:
This is a post submit job. So I guess optional or not does not really matter?